docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -119,6 +119,43 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+.. _postgresql-sql-support:
+
+SQL support
+-----------
+
+The connector provides **read access and limited write access** to data and
+metadata in PostgreSQL. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+
+    No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/select` including:
+
+  * :doc:`/sql/match-recognize`
+  * :doc:`/sql/values`
+
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/describe`
```

Comment:
if you order these "hierarchically" you would put SHOW COLUMS after SHOW TABLES

if you order alphabetically, you would put DESCRIBE before all SHOW statements.

please clarify intended ordering

Reply:
An oversight on my part. This is an alphabetical list.